tests/Microsoft.Bot.Builder.Tests/Teams/TeamsActivityHandlerHidingTests.cs

```diff
@@ -432,7 +432,7 @@ public async Task TestDelegatingTurnContext()
             // Arrange
             var turnContextMock = new Mock<ITurnContext>();
             turnContextMock.Setup(tc => tc.Activity).Returns(new Activity { Type = ActivityTypes.Message });
-            turnContextMock.Setup(tc => tc.Adapter).Returns(new BotFrameworkAdapter(new SimpleCredentialProvider()));
+            turnContextMock.Setup(tc => tc.Adapter).Returns(new NotImplementedAdapter());
```

Comment:
what does this test do?

Reply:
Looks like this test is verifying that `IBot.OnTurnAsync()` call will access the `turnContext.Adapter` property once.